src/doom/p_maputl.c

```diff
@@ -506,27 +507,28 @@ P_BlockLinesIterator
 //
 boolean
 P_BlockThingsIterator
-( int			x,
-  int			y,
```

Comment:
Why all these white space changes?

Reply:
It wasn't in accordance with the coding style (spaces only) so since I was going to change this function I fixed it. `clang-format` still isn't happy but we can't actually codify our style in its configuration.